Bootstrap/Bootstrap-icons.svg

```diff
@@ -0,0 +1,4066 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="svg1479"
+   sodipodi:docname="Bootstrap-icons.svg"
+   inkscape:version="0.92.4 (unknown)">
+  <title
+     id="title102576">Bootstrap Icons</title>
+  <metadata
+     id="metadata1485">
+    <rdf:RDF>
+      <cc:Work
+         rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title>Bootstrap Icons</dc:title>
+        <dc:date>04 Feb 2020</dc:date>
+        <dc:creator>
+          <cc:Agent>
+            <dc:title>meet</dc:title>
+          </cc:Agent>
+        </dc:creator>
+        <dc:rights>
+          <cc:Agent>
+            <dc:title>Custom</dc:title>
+          </cc:Agent>
+        </dc:rights>
+        <dc:publisher>
+          <cc:Agent>
+            <dc:title>Twitter Bootstrap</dc:title>
+          </cc:Agent>
+        </dc:publisher>
+        <dc:source>https://github.com/twbs/icons/releases/tag/v1.0.0-alpha2</dc:source>
+        <dc:subject>
+          <rdf:Bag>
+            <rdf:li>Inkscape</rdf:li>
+            <rdf:li>symbol</rdf:li>
+          </rdf:Bag>
+        </dc:subject>
+        <dc:description>Inkscape symbol pack based on Bootstrap icons</dc:description>
+        <dc:contributor>
+          <cc:Agent>
+            <dc:title>meet</dc:title>
+          </cc:Agent>
+        </dc:contributor>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <defs
+     id="defs1483" />
+  <sodipodi:namedview
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1"
+     objecttolerance="10"
+     gridtolerance="10"
+     guidetolerance="10"
+     inkscape:pageopacity="0"
+     inkscape:pageshadow="2"
+     inkscape:window-width="1366"
+     inkscape:window-height="706"
+     id="namedview1481"
+     showgrid="false"
+     inkscape:showpageshadow="false"
+     showborder="false"
+     inkscape:zoom="2.36"
+     inkscape:cx="-57.838983"
+     inkscape:cy="50"
+     inkscape:window-x="0"
+     inkscape:window-y="0"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg1479" />
+  <symbol
+     id="alarm-fill"
+     viewBox="0 0 20 20">
+    <path
+       fill-rule="evenodd"
```

Comment:
Try to remove this property. Sometimes is not required (is a common issue in figma-generated svgs)

Reply:
Ok, I will try and update.